plugins/generator-1.15.2/forge-1.15.2/procedures/entity_maketamed.java.ftl

```diff
@@ -0,0 +1,4 @@
+if ((${input$entity} instanceof TameableEntity)) {
```

Comment:
Please add a check if the source entity is an instance of PlayerEntity.

Reply:
It's done. The code is now `if ((${input$entity} instanceof TameableEntity) && (${input$sourceentity} instance of PlayerEntity)) {`